diagnostic.md

```diff
@@ -62,7 +62,7 @@ var result = strangeAdd(0, -5);
 
 What is the value of result?  Explain your answer.
 
-_Replace this text with your answer._
+- 5, strangeAdd a=0 and b= -5 since b makes the or of b> 0 it will return at A + B ( 0 + (-5)) = -5
```

Comment:
`result` equals 5. `a` is not greater than 0, nor is `b` greater than 0, therefore a-b = 5.